src/main/scala/io/iohk/ethereum/mallet/common/Util.scala

```diff
@@ -0,0 +1,30 @@
+package io.iohk.ethereum.mallet.common
+
+import java.io.{PrintWriter, StringWriter}
+import scala.reflect.runtime.universe._
+import scala.reflect.runtime.currentMirror
+
+object Util {
+
+  def sealedDescendants[T: TypeTag]: Set[T] = {
+    val symbol = typeOf[T].typeSymbol
+    val internal = symbol.asInstanceOf[scala.reflect.internal.Symbols#Symbol]
+    val descendants = internal.sealedDescendants.map(_.asInstanceOf[Symbol]) - symbol
+    descendants.map { d =>
+      val module = d.owner.typeSignature.member(d.name.toTermName)
+      currentMirror.reflectModule(module.asModule).instance.asInstanceOf[T]
+    }
+  }
+
+  def exceptionToString(ex: Throwable): String = {
+    val sw = new StringWriter()
+    sw.append(ex.getMessage + "\n")
+    ex.printStackTrace(new PrintWriter(sw))
+    sw.toString
+  }
+
+  implicit class OptionOps[T](opt: Option[T]) {
+    def toEither[U](left: U): Either[U, T] =
```

Comment:
There is `scala.Option.toRight`, so we can avoid introducing a new construct.

Reply:
lol, how did I miss that? :)